src/wms/src/main/java/org/geoserver/wms/map/PDFMapResponse.java

```diff
@@ -47,6 +70,11 @@
     /** A logger for this class. */
     private static final Logger LOGGER = org.geotools.util.logging.Logging
             .getLogger("org.vfny.geoserver.responses.wms.map.pdf");
+    
+    /**
+     * Whether to apply the new vector hatch fill optimization, or not (on by default, this is just a safeguard)
+     */
```

Comment:
the system property is in the method, not the javadoc.

You are correct that there is a lot of undocumented system properties in the codebase!  Tempted to make a foss4g presentation "geoserver terrors" to cover them all.

Please go ahead I will stop complaining about magic system properties.